frontend/src/views/Lobby.tsx

```diff
@@ -622,20 +665,56 @@ function LobbyPage() {
                   max={60}
                   value={duration}
                   disabled={!isHost(currentUser)}
+                  onChange={onSizeSliderChange}
+                  onMouseUp={updateRoomDuration}
+                />
+              </SliderContainer>
+            </HoverContainerSlider>
+            <NoMarginMediumText>Room Size</NoMarginMediumText>
+            <NoMarginSubtitleText>
+              {size === 31 ? 'No limit' : `${size === 1 ? '1 person' : `${size} people`}`}
```

Comment:
is this a template string inside a ternary inside a template string inside a ternary haha